app.js

```diff
@@ -29,8 +33,8 @@ function clientLoaded (client) {
   client.start('sla');
   client.on('StasisStart', function(event, channel) {
     if(!isDialed(event.args[0])) {
-      var bridgeName = event.args[0];
-      sla(client, channel, bridgeName)
+      var extension = event.args[0];
+      sla(client, channel, confFile, extension)
```

Comment:
Your parameter ordering here is a bit odd:

1: The ARI client (okay)
2: The channel entering the shared extension (passed in from Stasis, okay)
3: The config file for the shared extension (but what shared extension?)
4: The actual shared extension (passed in from Stasis...)

From an ease of use perspective, you should order your parameters such that they flow logically. One ordering would be:

```javascript
client, confFile, channel, extension
```

In other words: lead with the things that are unlikely to ever change (client, confFile) and follow with the more mutable pieces of data.